README.md

```diff
@@ -447,6 +448,37 @@ Approve a declarative migration plan.
 * `link` - The URL of the plan in [Atlas Registry](https://atlasgo.io/registry).
 * `status` - The status of the plan. For example, `PENDING` or `APPROVED`.
 
+### `ariga/atlas-action/monitoring`
+
+Takes snapshot of the database schema and uploads it to Atlas Cloud.
+Can be used periodically to [monitor](https://atlasgo.io/monitoring) changes in the database schema.
+
+#### Inputs
+
+* `cloud-token` - (required) The token that is used to connect to Atlas Cloud (should be passed as a secret).
+* `url` - (required) The URL of the database to take snapshot of. For example: `mysql://root:pass@localhost:3306/prod`.
+* `extID` - (optional) Unique identifier for the database server.
+* `schemas` - (optional) List of database schemas to include in snapshot (by default includes all schemas). see: https://atlasgo.io/declarative/inspect#inspect-multiple-schemas
+* `exclude` - (optional) List of exclude patterns from snapshot inspection. see: https://atlasgo.io/declarative/inspect#exclude-schemas
+
+#### Outputs
+
+* `snapshot-url` - The URL of the snapshot in Atlas Cloud.
+
+#### Example usage
+
+The next action will take a snapshot of the `dev` and `users` schemas of the `mysql://root:pass@localhost:3306` database.
+If the URL of the database changes, atlas knows to track that is the same database via the `extID` parameter.
+
+```yaml
+        uses: ariga/atlas-action/monitoring@v1
+        with:
+          cloud-token: ${{ secrets.ATLAS_CLOUD_TOKEN }}
+          url: 'mysql://root:pass@localhost:3306'
+          schemas: 'dev,users'
+          extID: 'dev-db'
+```
```

Comment:
Indention is weird here.

Reply:
this is the right Indention for step in github workflow yaml (so user can copy and paste)